Chap_API_Struct.tex

```diff
@@ -1431,6 +1487,78 @@ \subsubsection{Info structure list macros}
 
 Copy the provided key and data into a \refstruct{pmix_info_t} on the list. The key and any data stored in the source value can be modified or free'd without affecting the copied data once the function has completed.
 
+%%%%%%%%%%%
+\littleheader{Retrieve next \refstruct{pmix_info_t} on a list}
+\declareapiProvisional{PMIx_Info_list_get_info}
+
+%%%%
+\summary
+
+Retrieve next \refstruct{pmix_info_t} on a list.
+
+%%%%
+\format
+
+\versionMarker{5.0}
+\cspecificstart
+\begin{codepar}
+pmix_info_t *
+PMIx_Info_list_get_info(void *ptr,
+                        void *prev,
+                        void **next);
+\end{codepar}
+\cspecificend
+
+\begin{arglist}
+\argin{ptr}{A \code{void*} pointer initialized via \refapi{PMIx_Info_list_start} (handle)}
+\argin{prev}{Pointer to previous list element}
+\argout{next}{Pointer to next list element}
+\end{arglist}
+
+Returns \refconst{PMIX_SUCCESS} or a negative value corresponding to a PMIx error constant.
+
+%%%%
+\descr
+
+Retrieve the next info on the list. Passing NULL to the prev parameter will return the first pmix_info_t on the list. If next is set to NULL, it indicates the end of the list.
```

Comment:
Update this with new arg names prev->curr. "Retrieves current info and sets next pointer"